tests/us_enrichment/test_Client.js

```diff
@@ -0,0 +1,239 @@
+const chai = require("chai");
+const expect = chai.expect;
+const Client = require("../../src/us_enrichment/Client");
+const Lookup = require("../../src/us_enrichment/Lookup");
+const errors = require("../../src/Errors");
+const MockSender = require("../fixtures/mock_senders").MockSender;
+const MockSenderWithResponse = require("../fixtures/mock_senders").MockSenderWithResponse;
+const Attributes = require("../../src/us_enrichment/Response.js")
+const {Response, FinancialResponse, GeoResponse} = require("../../src/us_enrichment/Response");
+
+describe("A US Autocomplete Pro Client", function () {
```

Comment:
This is super minor, but do we want to change this to be US Enrichment?

Reply:
Good call. Got that changed